src/pudl/transform/eia860.py

```diff
@@ -312,12 +309,20 @@ def _core_eia860__generators(
             columns=["rto_iso_lmp_node_id", "rto_iso_location_wholesale_reporting_id"],
         )
         .pipe(pudl.helpers.convert_to_date)
-        .pipe(
-            pudl.metadata.classes.Package.from_resource_ids()
-            .get_resource("core_eia860__scd_generators")
-            .encode
-        )
     )
+    # Clean up a handful of non-standard BA codes prior to deduplication.
+    # This manual fix is required before encoding because there's not a unique mapping
+    # PA -> PACW in Oregon
+    gens_df.loc[
+        (gens_df.state == "OR") & (gens_df.balancing_authority_code_eia == "PA"),
+        "balancing_authority_code_eia",
+    ] = "PACW"
+    # PA -> PACE in Utah
+    gens_df.loc[
+        (gens_df.state == "UT") & (gens_df.balancing_authority_code_eia == "PA"),
+        "balancing_authority_code_eia",
+    ] = "PACE"
```

Comment:
Relocated from the `eia860m` specific transform to allow the spot fix to happen before the encoding in all cases.

Reply:
ah okay. so this was previously happening for 860m transform only because this code is not in the gens resource and therefor was not previously getting encoded. cool!

the first line of the comment does not make sense in this different context (the deduplication is an 860m only thing) and should be removed.